config/src/test/java/org/springframework/security/config/annotation/authentication/configuration/AuthenticationConfigurationTests.java

```diff
@@ -172,51 +171,49 @@ public AuthenticationManager authenticationManager() {
 			return this.authenticationManager;
 		}
 	}
-		//
```

Comment:
There seem to be some formatting changes in the code within this commit. While overall, they are a good thing I think we should move it to another PR or at minimum a separate commit. The reason is it helps us know the intent of the changes when being reviewed or when looking through commit history if we are trying to figure out a bug at some point.